rest-api/dao/participant_summary_dao.py

```diff
@@ -335,6 +335,15 @@ def to_client_json(self, model):
 
     return result
 
+  def _decode_token(self, query_def, fields):
+    decoded_vals = super(ParticipantSummaryDao, self)._decode_token(query_def, fields)
+    if query_def.order_by and (query_def.order_by.field_name == 'lastModified' and
```

Comment:
Let's add a comment describing why this logic is here.

Reply:
done.